File: backend/Marcador.Api/Models/MarcadorGlobal.cs
namespace Marcador.Api.Models
{
    public class MarcadorGlobal
    {public int Id { get; set; }

    public int EquipoLocalId { get; set; }
    public Equipo EquipoLocal { get; set; }

    public int EquipoVisitanteId { get; set; }
    public Equipo EquipoVisitante { get; set; }

    public int CuartoActual { get; set; } = 1;
    public int TiempoRestante { get; set; } = 600; 
    }
}

File: frontend/marcador-frontend/src/app/modelos/marcador-global.ts
import { Equipo } from './equipo';

export interface MarcadorGlobal {
  equipoLocal: Equipo;
  equipoVisitante: Equipo;
  cuartoActual: number;     // 1..4
  tiempoRestante: number;   // segundos (ej. 600)
  enProrroga: boolean;
  numeroProrroga: number;
}
